src/solver/simulation/common-eco-adq.cpp

```diff
@@ -190,7 +190,7 @@ bool ShouldUseQuadraticOptimisation(const Data::Study& study)
     if (!flowQuadEnabled)
         return false;
 
-    uint maxHours = study.runtime->nbHoursPerYear;
+    uint maxHours = 8760;
```

Comment:
We should not use hardcoded values for hoursPerYear, still need to figure out why 8760 and sometimes 8784
macro HOURS_PER_YEAR can be used instead of runtime